internal/oci/runtime_vm.go

```diff
@@ -164,9 +134,49 @@ func (r *runtimeVM) CreateContainer(ctx context.Context, c *Container, cgroupPar
 	}
 	r.Unlock()
 
+	return containerIO, nil
+}
+
+// CreateContainer creates a container.
+func (r *runtimeVM) CreateContainer(ctx context.Context, c *Container, cgroupParent string) (retErr error) {
+	log.Debugf(ctx, "RuntimeVM.CreateContainer() start")
+	defer log.Debugf(ctx, "RuntimeVM.CreateContainer() end")
+
+	// Lock the container
+	c.opLock.Lock()
+	defer c.opLock.Unlock()
+
+	// Lets ensure we're able to properly get construct the Options
```

Comment:
this reads to me more of an operation to do when creating the runtime. WDYt about moving it there?

Reply:
It seems ok, but this PR does n't change these lines.

Reply:
ah nevermind then

Reply:
This PR shuffles this code around, but this code was actually introduced to ensure we can configure the runtime to receive a specific configuration file.